frontend/src/app/shared/components/icon-picker/icon-picker-component.html

```
@@ -22,31 +22,30 @@ <h1>Choose an Icon</h1>
         }
       </p-tablist>
 
       <p-tabpanels>
-        <!-- Prime Icons Tab -->
         <p-tabpanel value="0">
           <div class="tab-content">
             <div class="search-container">
-              <i class="pi pi-search search-icon"></i>
+              <svg lucideSearch class="search-icon" aria-hidden="true"></svg>
               <input
                 type="text"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)